src/KBprojectorSparse.cc

```diff
@@ -103,7 +103,7 @@ void KBprojectorSparse::setNLindex(
 
     for (int i = 0; i < size_nl; i++)
     {
-        assert(i < lnumpt);
+        // assert(i < lnumpt);
```

Comment:
Should we just clean this out, so there's no question whether it should be there or not?

Reply:
I actually still have questions about those asserts, that's why I decided to keep them...